genienlp/data_utils/example.py

```diff
@@ -160,7 +164,7 @@ def from_examples(examples: Iterable[Example], numericalizer):
             all_context_plus_question_features.append(context_plus_question_feature)
 
         features = None
-        if args.do_ned and args.add_types_to_text == 'no' and args.add_qids_to_text == 'no':
+        if args.do_ned and args.add_entities_to_text == 'no':
             features = [a for a in all_context_plus_question_features if a]
```

Comment:
When is a Entity false-y? It's an object and it's always set, right? At best, it would be empty (pad entity) if there are no entities for a given token

Reply:
I remember it was possible before for question and context features to be empty but I can't find that code now. Perhaps it got refactored. I'll remove it then.